src/js/lib/monster.ui.js

```diff
@@ -1525,6 +1526,37 @@ define(function(require) {
 			return marked(content);
 		},
 
+		/**
+		 * Create a new instance of jsoneditor
+		 * @param {jQuery} $target Contatiner to append the editor to
+		 * @param {Object} [options] Editor options
+		 * @param {Object} [json] JSON object to set in the editor as initial data
+		 * @param {String} [customClass] Custom ccs class for the container
+		 * @return {JSONEditor} editor
+		 */
+		jsoneditor: function($target, options, json, customClass) {
+			if (!($target instanceof $)) {
+				throw TypeError('"$target" is not a jQuery object');
+			}
+
+			$target
+				.addClass(customClass ? customClass : 'monster-jsoneditor');
+
+			var container = $target[0],
+				jsonObject = json || {},
+				formattedOptions = _.merge({}, {
+					mode: 'code',
+					enableSort: false,
+					enableTransform: false,
+					search: true
```

Comment:
Could you explain why those should be set by default for every instance? (I'm curious and haven't read the editor's doc so just wondering).

Reply:
True that. I thought It would keep the editor as simple as possible, but I can rid off these options and it don't mean any significant impact in the editor